src/robot.cpp

```diff
@@ -100,23 +102,40 @@ class Robot
 {
     vector<Link> links;
     Integrator *joints;
+    vector<Vector> trajectory;
 
 public:
     Robot(const int n, const double length, const double Ts)
     {
         for (int i=0; i<n; i++)
             links.push_back(Link(length));
         joints=new Integrator(Ts,Vector(n,0.0));
+        trajectory = {};
     }
 
-    void move(const Vector &velocity, ImageOf<PixelRgb> &img)
+    void move(const Vector &velocity, ImageOf<PixelRgb> &img, bool visu_on)
     {
         Matrix H=eye(3,3);
         joints->integrate(velocity);
 
         cv::Mat imgMat=toCvMat(img);
         for (int i=0; i<links.size(); i++)
             H=links[i].draw(H,joints->get()[i],imgMat);
+        if (visu_on) {
+            if (trajectory.size() >= VEC_SIZE) {
+                trajectory.erase(trajectory.begin(),trajectory.begin()+VEC_SIZE/100);
+                yWarning("1/100 of trajectory data were removed to prevent excessive vector growth!");
```

Comment:
```suggestion
                yWarning()<<"1/100 of trajectory data were removed to prevent excessive vector growth!";
```
just to be consistent w/ the macro `yError()` already used.